projects/views.py

```diff
@@ -207,7 +211,23 @@ def delete_environment(request, project_slug):
         pk = request.POST.get("environment_pk")
         # TODO: Check that the user has permission to delete this environment.
         environment = Environment.objects.get(pk=pk, project=project)
-        environment.delete()
+
+        can_environment_be_deleted = True
```

Comment:
It would be nice with unit tests for the can x be deleted logic.

Reply:
I will take this commit out into a separate PR.